com.unity.shadergraph/Editor/ShaderFoundry/Linker/Properties/PropertyDeclarations.cs

```diff
@@ -420,12 +421,48 @@ static void BuildSimpleProperty(FieldPropertyContext context, string displayType
             MaterialPropertyDeclarationData.BuildSimple(context, displayType, defaultValue, resultProperty);
         }
 
-        static string GetDisplayType(FieldPropertyContext context, string displayType, HashSet<string> allowedDisplayTypeOverrides)
+        static string GetIntegerDisplayType(FieldPropertyContext context)
         {
             foreach (var attribute in context.Attributes)
             {
-                if (allowedDisplayTypeOverrides.Contains(attribute.Name))
+                if (attribute.Name == "Int" || attribute.Name == "Float")
```

Comment:
Is it ok to use string literals here?  Are they guaranteed to be spelled like this?  Could we not see "int" or "float" as well?

Reply:
It depends on if we consider our attributes to be case sensitive. Currently they are, so you can say:
`[Property][Int] float MyValue` but not `[int]`. I think this is common for attributes though.
We may consider making this a string in some shared location at some point though.